pages/linux/hdparm.md

```diff
@@ -0,0 +1,27 @@
+# hdparm
+
+> Get/set SATA/IDE device (HDD) parameters.
+
+- Request identification info of the device directly from the drive:
+
+`sudo hdparm -I /dev/{{device}}`
+
+- Get Advanced Power Management feature, if the drive supports it:
+
+`sudo hdparm -B /dev/{{device}}`
+
+- Set Advanced Power Management value. Values 1-127 permit spin-down, and values 128-254 do not:
```

Comment:
Perhaps this will read better?
```suggestion
- Set the Advanced Power Management value (values 1-127 permit spin-down, and values 128-254 do not):
```